client/fingerprint/env_aws_test.go

```diff
@@ -181,60 +220,133 @@ func startFakeEC2Metadata(t *testing.T) (endpoint string, cleanup func()) {
 type routes struct {
 	Endpoints []*endpoint `json:"endpoints"`
 }
+
 type endpoint struct {
-	Uri         string `json:"uri"`
-	ContentType string `json:"content-type"`
-	Body        string `json:"body"`
+	Uri         string
+	ContentType string
+	Body        string
```

Comment:
Apparently tags aren't needed, but didn't really need to remove them, can re-add them.